src/site/_includes/pattern-suite.njk

```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+---
+
+{{ content | safe }}
+
+<ul>
+{% for pattern in patterns %}
+  {% if pattern.suite == page.url %}
+    <li><a href="/patterns/{{pattern.id}}">{{ pattern.id }}</a></li>
```

Comment:
Could a pattern object have its own url as a property? Not necessary, just wondering if it would make sense.

Reply:
I think pattern id is a good approximation of a url. Also we collect patterns from files, so theorethically the do not have urls assigned to them yet (e.g. this depends on the eleventy build later on).